Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -60,6 +75,20 @@ const collectFilters = (params: IUrlParam[], possibleFilters: IFilter[]) => {
     return collectedFilters;
 };
 
+const collectSorting = (params: IUrlParam[], possibleSorting: ISort []) => {
```

Comment:
I see almost no differences with `collectFilters`. The most obvious difference are the types, i.e. `IFilter` and `ISort`, but they can be made a generic param.

Combine the two funcs